packages/@react-spectrum/picker/test/Picker.test.js

```diff
@@ -1085,7 +1085,8 @@ describe('Picker', function () {
       expect(picker).toHaveTextContent('Two');
     });
 
-    it('can select items with the Enter key', async function () {
+    // TODO: re-enable after merge
```

Comment:
?

Reply:
ah forgot to re-enable this lol, thanks for catching